frontend/src/tests/api.test.js

```diff
@@ -148,7 +146,4 @@ describe("Mock API tests", () => {
     describe("User permissions tests", () => {
         userPermissionsTests(mockAPI);
     });
-    describe("DDAH tests", () => {
-        ddahTests({ apiGET, apiPOST });
-    });
```

Comment:
These tests were failing for Mock API - deleted, since we don't intend to support all use cases for Mock API